specification/appconfiguration/AppConfiguration/main.tsp

```diff
@@ -0,0 +1,39 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "./routes.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+@service({
+  title: "Azure App Configuration",
+})
+@versioned(Versions)
+// FIXME: Check API key header name and find service scope
+@useAuth(
+  ApiKeyAuth<ApiKeyLocation.header, "Connection String"> | OAuth2Auth<[
+    {
+      type: OAuth2FlowType.implicit,
+      authorizationUrl: "https://login.microsoftonline.com/common/oauth2/authorize",
+      scopes: ["user_impersonation"],
```

Comment:
Also, do you know the authentication scope for your service? For example, the scope for cognitive services is `https://cognitiveservices.azure.com/.default`

Reply:
In your 1.0 spec I saw `https://azconfig.io/.default` is that the right one?

Reply:
This is cloud specific. For public clouds, `https://azconfig.io/.default` is correct. Alternatively, the AppConfig resource endpoint can also be used as scope. For eg: `https://contoso.azconfig.io/.default` is the scope for an AppConfig resource named `contoso`.

Reply:
Will leave the public cloud as the default then, folks who need a more specific scope can configure it on their end.